app/models/miq_expression.rb

```diff
@@ -1101,7 +1100,7 @@ def self.value2human(val, options = {})
       friendly = tables.split(".").collect do|t|
         if t.downcase == "managed"
           val_is_a_tag = true
-          @company + " Tags"
+          "#{User.current_tenant.name} Tags"
```

Comment:
I see where you're going here. But it will imply that tags are not shared across tenants when they are. I think it's ok but just wanted to make note of that.